src/Orchard.Web/Modules/Orchard.MessageBus/Services/SqlServerServiceBrokerFeatureGuard.cs

```diff
@@ -0,0 +1,103 @@
+using System;
+using System.Web.Mvc;
+using Orchard.Environment;
+using Orchard.Environment.Configuration;
+using Orchard.Environment.Extensions.Models;
+using Orchard.Environment.Features;
+using Orchard.Localization;
+using Orchard.Mvc;
+using Orchard.Mvc.Filters;
+using Orchard.UI.Notify;
+
+namespace Orchard.MessageBus.Services
+{
+    /// <summary>
+    /// Prevents the SQL Server Service Broker feature from being enabled for tenants that are not using
+    /// Microsoft SQL Server as their database provider.
+    /// </summary>
+    /// <remarks>
+    /// The implementation is hackish but there seems to be no other way: if it would use <see cref="Orchard.UI.Notify.INotifier"/>
+    /// it wouldn't work since <see cref="Orchard.UI.Notify.NotifyFilter"/> that writes out notifications to TempData 
+    /// runs before feature events are raised. Thus we have to manually add the messages to TempData, just as the filter 
+    /// would do.
+    /// </remarks>
```

Comment:
Namespace and rewrap:
```suggestion
    /// <summary>
    /// Prevents the SQL Server Service Broker feature from being enabled for tenants that are not using Microsoft SQL
    /// Server as their database provider.
    /// </summary>
    /// <remarks>
    /// The implementation is hackish but there seems to be no other way: if it would use <see cref="INotifier"/> it
    /// wouldn't work since <see cref="NotifyFilter"/> that writes out notifications to TempData runs before feature
    /// events are raised. Thus we have to manually add the messages to TempData, just as the filter would do.
    /// </remarks>
```